addons/web/static/src/legacy/utils.js

```diff
@@ -270,7 +270,14 @@ export function makeLegacyNotificationService(legacyEnv) {
                 }
             }
 
-            legacyEnv.services.notification = { notify, close };
+            function add(message, { title, type }) {
+                notification.add(message, {
+                    title,
+                    type,
+                });
+            }
+
+            legacyEnv.services.notification = { notify, close, add };
```

Comment:
I'm not sure I understand what is going on here. Aren't you supposed to use `notify` in this case ?

Reply:
@dmo-odoo Yes we could, but using `notify` will introduce a new traceback in To-Do because in that case we get `this.notification.add` method correctly.

Reply:
@aab-odoo If I understood the issue correctly, we have a component that is used both in legacy and in owl. Currently it does `this.notification.add` but it crashes in legacy so @maad-odoo here wanted to have `add` in the legacy env. I thought "but we can use notify", which is true but would require to detect if the env is legacy or not since the component can be used in both contexts. It would be nice to have the same call working in both cases, so what do you think of the suggested changes here ?